chapters/ch05.md

```diff
@@ -1,78 +1,99 @@
 # Your development and test sets
 
 ->
+# 5. Tập phát triển và tập kiểm tra
 
 Let’s return to our earlier cat pictures example: You run a mobile app, and users are uploading pictures of many different things to your app. You want to automatically find the cat pictures.
 
 ->
+Trở lại với ví dụ bức ảnh mèo ở phần trước: Khi bạn có một ứng dụng di động, và người dùng đang tải lên ảnh của nhiều thứ khác nhau đến ứng dụng của bạn. Bạn muốn tự động tìm ra đâu là các bức ảnh mèo.
+
 
 Your team gets a large training set by downloading pictures of cats (positive examples) and non-cats (negative examples) off of different websites. They split the dataset 70%/30% into training and test sets. Using this data, they build a cat detector that works well on the training and test sets.
 
 ->
 
+Nhóm của bạn có một tập dữ liệu lớn bằng cách tải các bức ảnh mèo (các mẫu dương) và các bức ảnh không có mèo (các mẫu âm) từ nhiều nguồn khác nhau. Tập dữ liệu này sau đó được chia 70%/30% thành tập huấn luyện và tập kiểm tra. Sử dụng tập dữ liệu này, bạn tạo ra một bộ nhận dạng mèo có thể hoạt động tốt ở cả tập huấn luyện và tập kiểm tra.
+
 But when you deploy this classifier into the mobile app, you find that the performance is really poor!
 
 ->
+Tuy nhiên, khi triển khai bộ nhận dạng mèo này lên một ứng dụng di động, bạn lại thấy rằng kết quả không thực sự tốt!
 
 ![img](../imgs/C05_01.png)
 
 ## What happened?
 
 ->
+## Điều gì đã xảy ra?
 
 You figure out that the pictures users are uploading have a different look than the website images that make up your training set: Users are uploading pictures taken with mobile phones, which tend to be lower resolution, blurrier, and poorly lit. Since your training/test sets were made of website images, your algorithm did not generalize well to the actual distribution you care about: mobile phone pictures.
 
 ->
+Bạn nhận ra rằng các bức ảnh được người dùng tải lên nhìn khác các bức ảnh mà bạn tải về từ các trang web cho tập dữ liệu của bạn: đó là vì các bức ảnh được chụp bằng điện thoại thường sẽ có độ phân giải thấp hơn, bị nhòe (mờ) hoặc bị tối hơn. Do bộ nhận dạng được huấn luyện trên tập dữ liệu từ ảnh trên các trang web nên nó không khái quát hóa tốt đến phân phối thực tế mà bạn cần nhắm đến, đó là ảnh chụp từ điện thoại.
```

Comment:
Câu này hơi lộn xộn. 
> Bạn nhận ra rằng các bức ảnh được người dùng tải lên nhìn khác các bức ảnh mà bạn tải về từ các trang web ~~cho tập dữ liệu của bạn~~- mà được dùng để xây dựng tập dữ liệu.

Reply:
>> ~~đó là vì các bức~~ do ảnh ~~được~ chụp bằng điện thoại thường ~~sẽ~~ có độ phân giải thấp hơn, bị nhòe (mờ) hoặc ~~bị~~ tối hơn

Reply:
- Em nghĩ nên dịch thẳng "thuật toán của bạn" 
> "nó"

Reply:
Đề xuất dịch câu cuối:
"Do tập dữ liệu huấn luyện và kiểm tra được tạo từ ảnh trên mạng, thuật toán của bạn không tổng quát hoá tốt tới nhóm ảnh bạn thực sự quan tâm: ảnh chụp từ điện thoại."

Reply:
Em thắc mắc là "nhóm ảnh bạn thực sự quan tâm" có thể hiện được ý nghĩa của từ "distribution" không ạ?

Reply:
Vì "tổng quát" = "khái quát" nên em nghĩ sẽ giữ nguyên là "khái quát" xuyên suốt cả bài luôn ạ.